docs/quickstart.rst

```diff
@@ -3,7 +3,7 @@ Quickstart
 
 **Globus Compute** client and endpoint software releases are available on `PyPI <https://pypi.org/project/funcx/>`_.
 
-You can try Globus Compute on a hosted Jupyter notebook with `Binder <https://mybinder.org/v2/gh/funcx-faas/examples/HEAD?filepath=notebooks%2FIntroduction.ipynb>`_
+You can try Globus Compute on a hosted `Jupyter notebook <https://jupyter.demo.globus.org/>`_
```

Comment:
Should we be more specific and point them to the folder/tutorial directly?  ie.

https://jupyter.demo.globus.org/hub/user-redirect/lab/tree/globus-jupyter-notebooks/Compute_Introduction.ipynb

(I'm not that familiar with notebooks, the base link just showed me a directory listing and I had to enter the directory and select a file)

Reply:
Hah! Ok then! I was told we couldn't do that. Nicely proven wrong. Thanks Lei!

Reply:
Fixed it now. Thanks!